docs/development/releasing.rst

```diff
@@ -228,6 +234,11 @@ packages that use the full bugfix/maintenance branch approach.)
       Also, it might be tempting to use the ``--tags`` argument to ``git push``,
       but this should *not* be done, as it might push up some unintended tags.
 
+.. _post-release-procedure:
+
+Post-Release procedures
```

Comment:
Is this where I should draw the line between "release" and "post-release"?

Reply:
pypi upload? after that the release cannot be modified? But honestly I don't really know.

Reply:
Okay. I moved post-release section to right after `twine upload`. The significance of this section separation is that in the RC instructions, it says "do not do the stuff in post-release".